wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -109,7 +109,9 @@ library
                      , aeson
                      , aeson-pretty
                      , async
+                     , base58-bytestring
```

Comment:
We need it because extended public key (received from Daedalus) will be in Base58-format.